Staging/AudioDxe/AudioDxe.h

```diff
@@ -119,4 +119,16 @@ extern
 UINTN
   gCodecSetupDelay;
 
+//
+// If to use forced codec
+//
+extern BOOLEAN
+  gForceCodec;
+
+//
+// Forced codec number
```

Comment:
Here and above, and below, project codestyle is to end comment text with full stop (aka period, US-eng).